docs/_build/html/_sources/compilers.rst.txt

```diff
@@ -15,3 +20,66 @@ Compilers for which ``xolotl`` has been known to work (or not).
  Ubuntu 17.10         g++          4.8                         ✓
  Manjaro Linux        g++          7.3                         ✓
 =================== ============ ============================ ======
+
+Compiling on Linux
+==================
+
+On certain Linux distributions (Arch-based in particular), ``MEX`` cannot identify
+the installed ``g++`` compiler, even when it exists on your path. The error
+looks something like this: ::
+
+  >> !which g++
+  /bin/g++
+  >> !g++ -dumpversion
+  8.1.0
+  >> mex test.cpp
```

Comment:
remove "mext test.cpp". the same thing can be achieved using "mex -setup C++" (later on here)

Reply:
done